Numsense/Numeral.fs

```diff
@@ -9,3 +9,6 @@ let tryParseEnglish = English.tryParseEnglishImp
 
 let toPolish =        Polish.toPolishImp
 let tryParsePolish =  Polish.tryParsePolishImp
+
+let toRussian =       Russian.toRussianImp Russian.Masculine
```

Comment:
From the `Russian.Masculine` argument passed here, I get the impression that it'd also be possible to convert to feminine or neuter. If so, is `toRussian` the appropriate name, or should it be `toRussianMasculine`? Or should we simply expose the function with that `Gender` argument?